src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,603 @@
+                Audiences = new string[] { "api://default" },
+            };
+            var testAuthorizationServerPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:{nameof(DeleteAuthorizationServer)}Test Authorization Policy",
+            };
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdAuthorizationPolicy = await testClient.AuthorizationServers.CreateAuthorizationServerPolicyAsync(testAuthorizationServerPolicy, createdAuthorizationServer.Id);
+            var retrievedAuthorizationPolicy = await testClient.AuthorizationServers.GetAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdAuthorizationPolicy.Id);
+            retrievedAuthorizationPolicy.Should().NotBeNull();
+
+            await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdAuthorizationPolicy.Id);
+            var ex = await Assert.ThrowsAsync<OktaApiException>(() => testClient.AuthorizationServers.GetAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdAuthorizationPolicy.Id));
+            ex.StatusCode.Should().Be(404);
+        }
+
+        [Fact]
+        public async Task ListOAuth2Scopes()
+        {
+            var testClient = TestClient.Create();
+            var defaultAuthorizationServer = await testClient.AuthorizationServers.ListAuthorizationServers().FirstOrDefaultAsync(authorizationServer => (bool)authorizationServer?.Name?.Equals("default"));
+
+            var allAuthorizationServerScopes = await testClient.AuthorizationServers.ListOAuth2Scopes(defaultAuthorizationServer.Id).ToListAsync();
+            allAuthorizationServerScopes.Should().NotBeNull();
+            allAuthorizationServerScopes.Count.Should().BeGreaterThan(0);
+        }
+
+        [Fact]
+        public async Task GetOAuth2Scope()
+        {
+            var testClient = TestClient.Create();
+            var defaultAuthorizationServer = await testClient.AuthorizationServers.ListAuthorizationServers().FirstOrDefaultAsync(authorizationServer => (bool)authorizationServer?.Name?.Equals("default"));
+            var oauth2Scope = await testClient.AuthorizationServers.ListOAuth2Scopes(defaultAuthorizationServer.Id).FirstOrDefaultAsync();
+
+            var retrievedOAuth2Scope = await testClient.AuthorizationServers.GetOAuth2ScopeAsync(defaultAuthorizationServer.Id, oauth2Scope.Id);
+            retrievedOAuth2Scope.Should().NotBeNull();
+            retrievedOAuth2Scope.Name.Should().Be(oauth2Scope.Name);
+        }
+
+        [Fact]
+        public async Task CreateOAuth2Scope()
+        {
+            var testClient = TestClient.Create();
+            var defaultAuthorizationServer = await testClient.AuthorizationServers.ListAuthorizationServers().FirstOrDefaultAsync(authorizationServer => (bool)authorizationServer?.Name?.Equals("default"));
```

Comment:
I know it's not possible to create our own AS today, but we shouldn't modify the default configuration. This can cause issues hard to identify.

Reply:
Good to know.